tests/link.js

```diff
@@ -139,7 +139,11 @@ describe( 'Link', () => {
 
 			linkButton.fire( 'execute' );
 
-			expect( selectUrlInputSpy.calledOnce ).to.true;
+			// Focus of url input is called async after internal promise resolve and we are
+			// not able to return this promise.
+			return wait().then( () => {
```

Comment:
It's an ugly hack, I don't like it but I didn't figure out anything less complicated.